zfs/ZfsCompressedArcMeter.h

```diff
@@ -11,7 +11,7 @@ in the source distribution for its full text.
 
 #include "Meter.h"
 
-void ZfsCompressedArcMeter_readStats(Meter* this, const ZfsArcStats* stats);
+extern void ZfsCompressedArcMeter_readStats(Meter* this, const ZfsArcStats* stats);
```

Comment:
Why is this changed to `extern`? This shouldn't be necessary for regular functions.